Data/Source/Scripts/AutoTest_Suite_NPCs.psc

```diff
@@ -54,9 +58,12 @@ endFunction
 ; Finalize the runs of tests
 ; [API] This function is optional
 function AfterTestsRun()
+  ; TODO: set PC Scale
   ConsoleUtil.ExecuteCommand("tai")
   ConsoleUtil.ExecuteCommand("tcai")
   ConsoleUtil.ExecuteCommand("tgm")
+  ; TODO enable UI

```

Comment:
It's not a `TODO` anymore: you just implemented it :D

Reply:
yeah, didn't clean up all the TODOs